site/using-weave/host-network-integration.md

```diff
@@ -38,6 +38,20 @@ Exposed addresses can also be added to weaveDNS by supplying fully qualified dom
     host2$ weave expose -h exposed.weave.local
     10.2.1.132
 
+###<a name="routing"></a>Routing from another host
+
+Having run `weave expose` as above, you can use Linux routing to give
+access to the Weave Network from hosts that are not running Weave Net:
+
+    ip route add <network-cidr> via <exposing-host>
+
+Where `<network-cidr>` is an IP address range in use on Weave Net,
+e.g. `10.2.0.0/16` or `10.32.0.0/12` and `<exposing-host>` is the
+address of the machine where you ran `weave expose`.
+
```

Comment:
Where,
 * `<network-cidr>` is an IP address range in use on Weave Net,
for example,  `10.2.0.0/16` or `10.32.0.0/12` and, 
 * `<exposing-host>` is the address of the machine on which you ran `weave expose`.